CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## XX.XX.XX
+* Added a new function "setID(newDeviceId)" for managing device id changes.

```

Comment:
+ according to the device ID Type

lets try this and mb we can change it later